src/app/entry/archive/dialog/archive-entry-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 mat-dialog-title>Are you sure you want to archive this Dockstore {{ term }}?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>
+    You are about to archive the {{ term }} <b>{{ path }}</b
+    >, which will make it read-only and less prominent in the Dockstore user interface. Specifically, after you archive the Dockstore
+    {{ term }}, it will be:
+  </p>
+
+  <ul>
+    <li>Marked as "Archived".</li>
+    <li>Read-only, except to unarchive.</li>
+    <li>Ranked lower in Dockstore Search.</li>
+    <li>Displayed in the "Archived" section of your Dockstore dashboard.</li>
+  </ul>
+  <p>Later, you can unarchive the {{ term }} to restore it to normal.</p>
```

Comment:
Wording suggestion: You can unarchive the <term> to restore it to normal at any time